src/React/DOM/Props.purs

```diff
@@ -297,6 +300,19 @@ radioGroup = unsafeMkProps "radioGroup"
 readOnly :: Boolean -> Props
 readOnly = unsafeMkProps "readOnly"
 
+ref :: String -> Props
+ref = unsafeMkProps "ref"
+
+-- | You can use `writeRef` to store a reference on `Refs`.
+-- | ``` purescrript
+-- | div [ refCb (writeRef this "inputEl") ] [...]
+-- | ```
+refCb
```

Comment:
I don't like this name so much, what about something like `withRef`?

Reply:
I actually I don't like it either, `withRef` sounds much better :)